articles/storage/blobs/storage-dotnet-how-to-use-blobs.md

```diff
@@ -20,385 +20,370 @@ ms.author: tamram
-* **Shared access signatures:** You can provide clients with a shared access signature (SAS), which provides delegated access to a resource in your storage account, with permissions that you specify and over an interval that you specify. See [Using Shared Access Signatures (SAS)](../common/storage-dotnet-shared-access-signature-part-1.md?toc=%2fazure%2fstorage%2fblobs%2ftoc.json) for more information.
+Finally, you can publish to Azure!
 
-### Encrypting blob data
-Azure Storage supports encrypting blob data both at the client and on the server:
+1. In **Solution Explorer**, right-click and select **Publish**.
+2. Under **Microsoft Azure App Service**, ensure that **Create New** is selected and press the **Publish** button.
+3. Fill out the form - all fields should be automatically populated with values that will allow you to create a new App Service web app with the project.
+4. Click **Create**.
 
-* **Client-side encryption:** The Storage Client Library for .NET supports encrypting data within client applications before uploading to Azure Storage, and decrypting data while downloading to the client. The library also supports integration with Azure Key Vault for storage account key management. See [Client-Side Encryption with .NET for Microsoft Azure Storage](../common/storage-client-side-encryption.md?toc=%2fazure%2fstorage%2fblobs%2ftoc.json) for more information. Also see [Tutorial: Encrypt and decrypt blobs in Microsoft Azure Storage using Azure Key Vault](storage-encrypt-decrypt-blobs-key-vault.md).
-* **Server-side encryption**: Azure Storage now supports server-side encryption. See [Azure Storage Service Encryption for Data at Rest (Preview)](../common/storage-service-encryption.md?toc=%2fazure%2fstorage%2fblobs%2ftoc.json).
+After it creates the App Service web app, and the application is fully published, a browser window will open and you can interact with the sample app live over the internet!
 
 ## Next steps
-Now that you've learned the basics of Blob storage, follow these links
-to learn more.
+
+Now that you've learned the basics of Blob storage, follow these links to learn more.
 
 ### Microsoft Azure Storage Explorer
-* [Microsoft Azure Storage Explorer (MASE)](../../vs-azure-tools-storage-manage-with-storage-explorer.md) is a free, standalone app from Microsoft that enables you to work visually with Azure Storage data on Windows, macOS, and Linux.
+* [Microsoft Azure Storage Explorer](https://docs.microsoft.com/azure/vs-azure-tools-storage-manage-with-storage-explorer) is a free, crossplatform application which allows you to work visually with Azure Storage Data.
 
-### Blob storage samples
-* [Getting Started with Azure Blob Storage in .NET](https://azure.microsoft.com/documentation/samples/storage-blob-dotnet-getting-started/)
+### More samples
```

Comment:
How about something in here about sample with other technologies? Xamarin, Windows, etc? I think this is a good hand off area.

Reply:
I'll take a look for more.

Reply:
Found some!